target_chains/solana/pyth_solana_receiver_state/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "pyth-solana-receiver-state"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["lib"]
+name = "pyth_solana_receiver_state"
+
+
+[dependencies]
+anchor-lang = ">=0.28.0"
```

Comment:
Here I can use a loose anchor so the consumer can pick whichever version they want